### PR TITLE
New convert intrinsic

### PR DESCRIPTION
**Pull request by gita-omr**
Depends on https://github.com/eclipse/omr/pull/6936

### CODE REVIEW

**Comment by gita-omr** [comment]
Fixed line endings. Please notice that I will improve PR description after I squash the commits. Please see commit descriptions for now.

**Comment by gita-omr** [comment]
Jenkins test sanity.openjdk all jdk19 depends https://github.com/eclipse/omr/pull/6936

**Comment by gita-omr** [comment]
Addressed test failures

**Comment by gita-omr** [comment]
Jenkins test sanity.openjdk all jdk19 depends https://github.com/eclipse/omr/pull/6936

File: runtime/compiler/optimizer/VectorAPIExpansion.cpp [hunk 1]
```diff
@@ -159,14 +231,52 @@ TR_VectorAPIExpansion::visitNodeToBuildVectorAliases(TR::Node *node)
...
+               {
+               // propagate vector info from VectorAPI call to temp
+               TR::DataType elementType;
+               int32_t numLanes;
+
+               getElementTypeAndNumLanes(rhs, elementType, numLanes);
+
+               if (getReturnType(rhs->getSymbolReference()->getSymbol()->castToMethodSymbol()) == Mask &&
+                   (elementType == TR::Float || elementType == TR::Double))
+                  elementType =( elementType == TR::Float) ? TR::Int32 : TR::Int64;
```
[the remark this answers is not in the document]
Comment (gita-omr) [reply runtime/compiler/optimizer/VectorAPIExpansion.cpp]: Fixed. Thanks!

**Comment by knn-k** [comment]
Jenkins test sanity.openjdk all jdk19 depends https://github.com/eclipse/omr/pull/6936

**Review by knn-k — APPROVED** [review]

**Pull request merged**